packages/vscode/src/spec.mts

```diff
@@ -3,13 +3,10 @@ import { parseStringPromise } from 'xml2js';
 import { GmlSpec, gmlSpecSchema } from './spec.schemas.mjs';
 export type { GmlSpec } from './spec.schemas.mjs';
 
-// @ts-expect-error
-import fallbackSpec from '../samples/GmlSpec.xml';
+let fallbackSpec = require('../samples/GmlSpec.xml');
```

Comment:
Can you explain why you switched this to a `require`?

Reply:
Using `require()` can ignore ESLint errors, although in fact I don't recommend doing this